data/json/overmap_specials.json

```diff
@@ -220,6 +220,20 @@
         "flags" : [ "CLASSIC" ]
     },{
         "type" : "overmap_special",
+        "id" : "Sugar House",
+        "overmaps" : [
+            { "point":[0,0,0], "overmap": "sugar_house" }
```

Comment:
What needs to be done here? Should I simply change this overmap value from `sugar_house` to `sugar_house_north`? Will I need to change it somewhere else?